tests/integration/model/modelController.spec.ts

```diff
@@ -369,4 +372,76 @@ describe('ModelController', function () {
       });
     });
   });
+
+  describe('DELETE /models/:identifier', function () {
+    describe('Happy Path 🙂', function () {
+      it('Should return 200 status code and the deleted model', async function () {
+        const expected = createFakeDeleteResponse();
```

Comment:
In other files you are setting the expected directly (like modelManager.spec.ts). You should be consistent

Reply:
is this is a better option ??

Reply:
I didn't provide you an option. I am just saying that you must be consistent. If you decide to make a constant with the response, use this constant in every function you need

Reply:
ok